eclass/kernel-build.eclass

```diff
@@ -41,6 +41,50 @@ BDEPEND="
 	app-alternatives/yacc
 "
 
+IUSE="+strip"
+
+# @ECLASS_VARIABLE: ALLOW_MODULES_SIGN
+# @PRE_INHERIT
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# If set to a non-null value, adds IUSE=modules-sign and required
```

Comment:
Sigh, I wish I had noticed that grammar failure before it went public.

Reply:
:O where's the grammar mistake? The sentence reads fine to me but maybe I'm just reading over the mistake.

Reply:
We already had a global IUSE named "modules", variables and USE are meant as suffix to that.

Did talk about this a bit when picked it in the commit message, but nobody commented.

Reply:
It's "commit signing", not "commits signing". I'm not an expert on English but the latter sounds wrong.

Reply:
I see it more as group than something to read like a sentence, not to mention it says "sign" rather "signing".

like <group>-<action>

Reply:
aah now I see what you mean.

Personally, what I think would sound best is `sign-modules`. But well that would then break the sorting with the existing `modules` flag. 

In any case, it is what it is, lets just keep things as consistent as possible.

Reply:
What about ˋKERNEL_IUSE_MODULES_SIGNˋ? Bit on the long side, but it is descriptive.

Reply:
WFM, I guess.

Reply:
Great, Fixed